hack/kfc-dev.js

```diff
@@ -0,0 +1,119 @@
+// Helper for using a KFC release candidate in Pepr
+const { exec } = require("child_process");
+const fs = require("fs");
+const p = require("path");
+const { Command } = require("commander");
+
+const program = new Command();
+const repoUrl = `https://github.com/defenseunicorns/kubernetes-fluent-client.git`;
+const moreInfo = "\n\nUse node hack/kfc-dev.js --help for more information.";
+
+// Helper function to run shell commands
+const runCmd = (cmd, dir) => {
+  return new Promise((resolve, reject) => {
+    exec(cmd, { cwd: dir }, (error, stdout, stderr) => {
+      if (error) {
+        console.error(`Error: ${error.message}`);
+        reject(error);
+        process.exit(1);
+        return;
+      }
+      if (stderr) {
+        console.log(stderr);
+      }
+      console.log(stdout);
+      resolve();
+    });
+  });
+};
+
+const runSequence = async (...commands) => {
+  try {
+    for (const { cmd, dir } of commands) {
+      await runCmd(cmd, dir);
+    }
+    console.log("All commands executed successfully.");
+  } catch (error) {
+    console.error("An error occurred during command execution:", error);
+  }
+};
+
+// Command Definitions
+program
+  .name("kfc-dev")
+  .description("Helper for using a KFC release candidate in Pepr.")
+  .version("0.0.1");
+
+// Build Command
+program
+  .command("build")
+  .description("Builds a dev image from a repo or local path")
```

Comment:
```suggestion
  .description("Builds a custom dev image")
```
Still had the "from a repo" stuff in it which made it kinda seem like a `remote vs. local` thing (which it isn't, of course).  Seems the short form is plenty understandable too though, especially since the options describe themselves well now.